nearswap/src/lib.rs

```diff
@@ -817,17 +819,19 @@ mod tests {
 
     #[test]
     fn add_liquidity_happy_path() {
-        let (mut ctx, mut c) = init();
+        let ynear_deposit = 3 * NDENOM;
+        let token_deposit = 1 * NDENOM;
+        let ynear_deposit_with_storage = ynear_deposit + NEP21_STORAGE_DEPOSIT;
+
+        let (mut ctx, mut c) = _init(ynear_deposit_with_storage);
         let t = ctx.accounts.token1.clone();
         let a = ctx.accounts.predecessor.clone();
 
         // in unit tests we can't do cross contract calls, so we can't check token1 updates.
         check_and_create_pool(&mut c, &t);
 
-        let ynear_deposit = 30 * NDENOM;
-        let token_deposit = 10 * NDENOM;
-        let ynear_deposit_with_storage = ynear_deposit + NEP21_STORAGE_DEPOSIT;
-        ctx.set_deposit(ynear_deposit_with_storage);
+        // Problem: Setting value directly in ctx results in failure of test
+        //ctx.set_deposit(ynear_deposit_with_storage);
```

Comment:
the right way is to use `test!` makro every time we update the context.

Reply:
what do you mean ?
put `test!` after ctx.set_deposit() ?

Reply:
or instead of `#[test]` put `#[test!]` this also doesn't work

Reply:
Sorry, I meant `testing_env!` macro. Check [here](https://github.com/robert-zaremba/vostok-dao/blob/master/src/lib.rs#L234) or [here](https://github.com/ref-finance/ref-contracts/blob/main/ref-exchange/src/lib.rs#L243).

Reply:
ok got it, i updated it using _init() function which uses testing_env!